test/src/e2e_vm_tests/test_programs/should_pass/dca/impl_unused_fn/src/main.sw

```diff
@@ -0,0 +1,20 @@
+script;
+
+struct Data1 {
+    value: u64
+}
+
+impl Data1 {
+    fn get(self) -> u64 {
+        self.value
+    }
+
+
+    fn get2(self) -> u64 {
+        self.value
+    }
```

Comment:
Can we add a third method here that is actually used to make sure that we're not emitting warnings for everything in the `impl` block?